package.json

```diff
@@ -67,7 +67,7 @@
     "decomment": "^0.9.1",
     "del": "^3.0.0",
     "dialog-polyfill": "^0.4.10",
-    "generator-pfelement": "^1.4.0",
```

Comment:
@wesruv I don't think we want to remove the generator as a depedency.

Reply:
@kylebuch8 Fixed